src/Entity/User.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+class User {
+
+    /**
+     *
+     * @var string
+     */
+    private $hashedPassword;
+
+    /**
+     *
+     * @var string 
+     */
+    private $salt;
+
+    /**
+     *
+     * @var integer
+     */
+    private $hashTimes;
+
+    /**
+     * 
+     * @param string $hashedPassword
+     * 
+     */
+    public function setHashedPassword($hashedPassword) {
+        $this->hashedPassword = $hashedPassword;
```

Comment:
https://stackoverflow.com/questions/11072965/why-return-this-in-setter-methods
Paskaityk (labiau antras atsakymas man patinka) ir padaryk; prieš antrą ir paskesnes `->` reiktų naujos eilutės ir tabuliavimo tinkamo (pažiūrėk, gal tau ide pasiūlys iškart grąžų variantą)

Reply:
Sveikas, ką paskaičiau viskas aišku ir logiška, tiesa iškilo klausimas dėl tokios funkcijos aprašymo, ties _@return type $this__  eilute, kaip reikėtų apsirašyt tą $this, nors aš, spėčiau šioje situacijoje tos eilutės, turbūt, net ir nereikia ?

Reply:
@return reikia. Šioje klasėje galima arba @return $this arba @return User. Tarp šių variantų yra šioks toks skirtumas, bet su dabartiniu projektu mes jo nepamatysim...